packages/yoroi-extension/chrome/manifest.template.js

```diff
@@ -66,15 +66,15 @@ export default ({
     permissions: [
       'storage',
       '*://connect.trezor.io/*',
-      'https://emurgo.github.io/yoroi-extension-ledger-connect/*'
+      'https://emurgo.github.io/yoroi-extension-ledger-connect-vnext/*'
```

Comment:
I'm hoping that since this is the same domain as the previous URL that it doesn't re-prompt users to accept this extension's permission :/
We have no good way to version this library, so deploying to a different URL is the only clean way to do it.